Documentation/tuning.md

```diff
@@ -23,10 +23,10 @@ The default values can be overridden on the command line:
 
 ```sh
 # Command line arguments:
```

Comment:
Seems like the intention is 10x election timeout can be overridden.

> The default values can be overridden on the command line:

Reply:
ok sounds good. feel free to close it. opened it as it was confusing to me

Reply:
Yeah let me close it. Please let us know if you have any other questions.
Thanks!